.circleci/config.yml

```diff
@@ -574,76 +627,149 @@ jobs:
       - slack/status:
           webhook: << parameters.slack-webhook >>
 
+  desktop-build-source:
+    <<: *desktop_defaults
+    steps:
+      - checkout
+      - run: *update-node
+      - run: *yarn-install
+      - restore_cache: *desktop-restore-source-cache
+      - run: *desktop-decrypt-assets
+      - run:
+          name: Build Desktop Source
+          no_output_timeout: 15m
+          command: |
+            make -f desktop/Makefile source
+      - save_cache: *desktop-save-source-cache
+      - persist_to_workspace:
+          root: /home/circleci/wp-calypso
+          paths: *desktop-cache-paths
+
+  desktop-build-mac:
+    macos:
+      xcode: "10.1.0"
+    shell: /bin/bash --login
+    working_directory: /Users/distiller/wp-calypso
+    environment:
+      DETECT_CHROMEDRIVER_VERSION: 'false'
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /Users/distiller/wp-calypso
+      - run: *update-node
+      - run: *yarn-install
+      - run:
+          name: Build Desktop Mac
+          no_output_timeout: 30m
+          environment:
+            CSC_LINK: resource/certificates/mac.p12
+            CSC_FOR_PULL_REQUEST: true
+          command: |
+                  set +e
+                  source $HOME/.nvm/nvm.sh
+                  nvm use
+
+                  make -f desktop/Makefile package
+      - run: *desktop-artifacts-cleanup
+      - run: *move-desktop-mac-artifacts
+      - persist_to_workspace:
+          root: ~/wp-calypso
+          paths:
+            - desktop/release
+
+  desktop-artifacts:
+    docker:
+      - image: buildpack-deps:trusty
+    working_directory: /wp-desktop
+    steps:
+      - attach_workspace:
+          at: /wp-desktop
+      - run:
+          name: Clean Up
+          command: |
+            find release -type f -name "._*" -delete
+            rm -rf release/.icon-set
+      - persist_to_workspace:
+          root: /wp-desktop
+          paths:
+            - release
+      - store_artifacts:
+          path: release
+
 workflows:
   version: 2
   calypso:
     jobs:
-      - setup
-      - prime-calypso-live:
-          filters:
-            branches:
-              ignore: master
-      - test-full-site-editing:
-          requires:
-            - setup
-      - build-notifications:
-          requires:
-            - setup
-      - build-o2-blocks:
-          requires:
-            - setup
-      - build-wpcom-block-editor:
-          requires:
-            - setup
-      - lint-and-translate:
-          requires:
-            - setup
-      - test-client:
+#       - setup
```

Comment:
Temporarily commenting out to speed up the feedback loop while we're iterating on this.